docs/_docs/github-pages.md

```diff
@@ -5,102 +5,80 @@ permalink: /docs/github-pages/
 
 [GitHub Pages](https://pages.github.com) are public web pages for users,
 organizations, and repositories, that are freely hosted on GitHub's
-`github.io` domain or on a custom domain name of your choice. GitHub Pages are
-powered by Jekyll behind the scenes, so in addition to supporting regular HTML
-content, they’re also a great way to host your Jekyll-powered website for free.
+`github.io` domain or on a custom domain name of your choice. GitHub Pages are powered by Jekyll behind the scenes, so they're a great way to host your Jekyll-powered website for free.
+
+Your site is automatically generated by GitHub Pages when you push your source
+files. Note that GitHub Pages works equally well for regular HTML content,
+simply because Jekyll treats files without YAML front matter as static assets.
+So if you only need to push generated HTML, you're good to go without any further setup.
 
 Never built a website with GitHub Pages before? [See this marvelous guide by
-Jonathan McGlone to get you up and running](http://jmcglone.com/guides/github-pages/).
-This guide will teach you what you need to know about Git, GitHub, and Jekyll to create your very own website on GitHub Pages.
+Jonathan McGlone to get you up and
+running](http://jmcglone.com/guides/github-pages/). This guide will teach you
+what you need to know about Git, GitHub, and Jekyll to create your very own
+website on GitHub Pages.
+
+##  The github-pages gem
+
+Our friends at GitHub have provided the
+[github-pages](https://github.com/github/pages-gem) gem which is used to manage
+[Jekyll and its dependencies on GitHub Pages](https://pages.github.com/versions/).
+Using it in your projects means that when you deploy your site to GitHub Pages,
+you will not be caught by unexpected differences between various versions of the
+gems.
+
+GitHub Pages runs in `safe` mode and only allow [a set of whitelisted plugins](https://help.github.com/articles/configuring-jekyll-plugins/#default-plugins),
```

Comment:
s/allow/allows

Reply:
I wish we had [Sedy](https://marmelab.com/sedy/) on the repo 😄

Reply:
split line at around 80 chars..

Reply:
^ that too